api/proto/teleport/legacy/types/types.proto

```diff
@@ -4429,8 +4429,13 @@ message DatabaseServiceV1 {
 // DatabaseServiceSpecV1 is the DatabaseService Spec.
 message DatabaseServiceSpecV1 {
   // ResourceMatchers is the configured match for Database resources.
-  repeated wrappers.LabelValues ResourceMatchers = 1 [
-    (gogoproto.jsontag) = "resources",
+  repeated ResourceMatcher ResourceMatchers = 1 [(gogoproto.jsontag) = "resources"];
+}
+
+// ResourceMatcher is a set of properties that is used to match on resources.
+message ResourceMatcher {
+  wrappers.LabelValues labels = 1 [
```

Comment:
```suggestion
  wrappers.LabelValues Labels = 1 [
```
?

Also, do you think if `(gogoproto.nullable) = false, ` is needed for either of these?

Reply:
My 2c: don't use gogo features unless you absolutely need them, it's deprecated and we would like to move out of it sometime.